### ISSUE TITLE
lighthouse issues

### ISSUE THREAD

**Issue by ahmedkaludi** [ahmedkaludi/accelerated-mobile-pages#3547]
- [x] 1. Buttons do not have an accessible name
When a button doesn’t have an accessible name, screen readers announce it as “button”, making it unusable for users who rely on screen readers

Failing Elements
<button on=”tap:search-icon”> <i class=”icono-search”></i> </button>
screenshot http://prntscr.com/op5uab

- [x] 2. i have alt tag for images but it not shown and error like ” ”

Screenshot http://prntscr.com/op5v1a

- [x] there are other errors shown in the above screenshot which we need to fix as well.

The page I need help with: https://jobdashboard.in/10th-12th-govt-jobs-updates-2019-2020-central/amp/

REF: https://wordpress.org/support/topic/reg-error-buttons-do-not-have-an-accessible-name/

**Comment by ahmedkaludi** [comment]
- [ ]  Need to fix these issues reported on https://lighthouse-dot-webdotdevsite.appspot.com/lh/html?url=https://jobdashboard.in/upmscl-jobs-2019-150-junior-pharmacist-posts/?amp#performance 
 REF:  https://wordpress.org/support/topic/lighthouse-issuse/#post-11842319

**Comment by ahmedkaludi** [comment]
Also test the Single post and Homepage with

- [ ] Design 1

- [ ] Design 2

- [ ] D3

- [ ] Swift

**Comment by MohammedKaludi** [comment]
@ahmedkaludi 

TEST URL - https://jobdashboard.in/upmscl-jobs-2019-150-junior-pharmacist-posts/?amp

For breadcrumbs red color
        #f42f42 - now
	#AD0B15 - new

For grey color
#acacac - now
#575757 - new - for .breadcrumb .bread-post and .amp-wp-meta, .amp-wp-meta a

**Comment by MohammedKaludi** [comment]
The way to fetching alt is incorrect. 
https://github.com/ahmedkaludi/accelerated-mobile-pages/commit/62375adfde562be2a51e5e187a6f6759e181683d#diff-6aa5512f7dd8d5ce60fc97b0f340912cR6813

**Comment by MohammedKaludi** [comment]
This might  break the comments jump link 
https://github.com/ahmedkaludi/accelerated-mobile-pages/commit/291749ade3652fc98f67a7fc4458bc76ce7a2678#diff-6aa5512f7dd8d5ce60fc97b0f340912cR2207

**Comment by MohammedKaludi** [comment]
Ready to push, but do not close the ticket after the update.

**Comment by Zabi09** [comment]
Pushed in 0.9.98.8
